src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -75,7 +93,12 @@ public Double averageSumOdd(List<Integer> numbers) {
      **/
     public List<People> workablePeople(int fromAge, int femaleToAge,
                                        int maleToAge, List<People> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter((p) -> p.getAge() >= fromAge)
+                .filter((p) -> (p.getSex() == People.Sex.WOMEN
```

Comment:
1) we don't need brackets when we have only one parameter for lambda;
2) do we need to use two filters?

Reply:
done